### ISSUE TITLE
Wording for locking error messages

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#16983]
From @jancborchardt :
>Maybe we should reword it to "is in use"? Both locked and busy do not fully convey what is happening.
(https://github.com/owncloud/core/pull/16892#issuecomment-112726809)

I guess that might do. I can change the message in LockingException then.

@icewind1991 @DeepDiver1975 @nickvergessen

**Comment by PVince81** [comment]
Now I also realize that this exception is not being translated either, but will be shown as is in the UI.

If we want a proper translated message, this means again catching the exception explicitly everywhere (all ajax code paths) and translating it when returning the error message to the client.

**Comment by PVince81** [comment]
and this effort will be nullified again once we move the files app to WebDAV

**Comment by nickvergessen** [comment]
I'd say not translating is okay for now. Translators would still need to update their translations within the next days to get it in the package, so ...

**Comment by DeepDiver1975** [comment]
> and this effort will be nullified again once we move the files app to WebDAV

won't fix then?
